packages/replay/src/types.ts

```diff
@@ -112,6 +112,13 @@ export interface ReplayPluginOptions extends SessionOptions {
    * Block all media (e.g. images, svg, video) in recordings.
    */
   blockAllMedia: boolean;
+
+  /**
+   * _experiments allows the user to send options to define how this integration works.
+   *
+   * Default: undefined
+   */
+  _experiments?: Partial<{ captureExceptions: boolean }>;
```

Comment:
l: let's make it clear that this object controls experimental/internal features

```suggestion
  /**
   * _experiments allows users to enable experimental or internal features.
   * We don't consider such features as part of the public API and hence we don't guarantee semver for them.
   * Experimental features can be added, changed or removed at any time.
   *
   * Default: undefined
   */
  _experiments?: Partial<{ captureExceptions: boolean }>;
```